runtime/src/bank.rs

```diff
@@ -626,6 +626,18 @@ impl TransactionExecutionResult {
     }
 }
 
+pub struct LoadAndExecuteTransactionsOutput {
+    pub loaded_transactions: Vec<TransactionLoadResult>,
+    // Vector of results indicating whether a transaction was executed or could not
+    // be executed. Note executed transactions can still have failed!
+    pub execution_results: Vec<TransactionExecutionResult>,
+    pub retryable_transaction_indexes: Vec<usize>,
+    // Total number of transactions that were executed
+    // Note executed transactions can still have failed!
+    pub executed_transactions_count: usize,
```

Comment:
Heh, actually this tx count has always been wrong. It only counts successfully executed transactions, it doesn't include failed transactions even if they can be committed.